mappings/net/minecraft/world/loot/LootTableReporter.mapping

```diff
@@ -2,7 +2,12 @@ CLASS cof net/minecraft/world/loot/LootTableReporter
 	FIELD a messages Lcom/google/common/collect/Multimap;
 	FIELD b nameFactory Ljava/util/function/Supplier;
 	FIELD c name Ljava/lang/String;
+	METHOD <init> (Lcom/google/common/collect/Multimap;Ljava/util/function/Supplier;)V
+		ARG 1 rmessages
```

Comment:
`reportedMessages`?

Reply:
No, sometimes when I press `r` to rename, Enigma puts a `r` in the rename field while doing so. That's an instance of when I missed that happening.